apps/institution/mutations.py

```diff
@@ -0,0 +1,88 @@
+import graphene
+from django.utils.translation import ugettext as _
+
+from utils.graphene.mutation import generate_input_type_for_serializer
+from utils.graphene.error_types import CustomErrorType, mutation_is_not_valid
+
+from apps.institution.models import Institution
+from apps.institution.schema import InstitutionType
+from apps.institution.serializers import (
+    InstitutionSerializer,
+)
+
+
+InstitutionInputType = generate_input_type_for_serializer(
+    'InstitutionCreateInputType',
+    serializer_class=InstitutionSerializer
+)
+
+
+class CreateInstitute(graphene.Mutation):
+    class Arguments:
+        data = InstitutionInputType(required=True)
+
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    ok = graphene.Boolean()
+    result = graphene.Field(InstitutionType)
+
+    @staticmethod
+    def mutate(root, info, data):
+        serializer = InstitutionSerializer(data=data, context={'request': info.context.request})
+        if errors := mutation_is_not_valid(serializer):
+            return CreateInstitute(errors=errors, ok=False)
+        instance = serializer.save()
+        return CreateInstitute(result=instance, errors=None, ok=True)
```

Comment:
Let's create a base Mutation for Create and Update Mutations.

This can be generalized. For ref look in deep.
```python3
    @classmethod
    def mutate(cls, root, info, data):
        serializer = cls.serializer(data=data, context={'request': info.context.request})
        if errors := mutation_is_not_valid(serializer):
            return cls(errors=errors, ok=False)
        instance = serializer.save()
        return cls(result=instance, errors=None, ok=True)
```